modules/sops/templates/default.nix

```diff
@@ -52,18 +52,32 @@ in
                 '';
               };
               owner = mkOption {
-                type = types.singleLineStr;
-                default = "root";
+                type = with lib.types; nullOr singleLineStr;
+                default = null;
+                description = ''
+                  User of the file. Can only be set if uid is 0;
+                '';
+              };
+              uid = mkOption {
```

Comment:
Can you also add those to `sops-nix/modules/nix-darwin/default.nix`?

Reply:
No problem!